ska_sun/sun.py

```diff
@@ -11,19 +14,18 @@
 from chandra_aca.transform import radec_to_eci
 from Quaternion import Quat
 from ska_helpers import chandra_models
-from ska_helpers.utils import LazyVal
 
-CHANDRA_MODELS_PITCH_ROLL_FILE = "chandra_models/pitch_roll/pitch_roll_constraint.csv"
 
+@functools.lru_cache()
+def _get_roll_table(repo_dir, version):
+    model_path = Path("chandra_models") / "pitch_roll" / "pitch_roll_constraint.csv"
 
-def load_roll_table():
-    """Load the pitch/roll table from the chandra_models repo."""
-
+    # Read in data from `filename`, basically the existing load table function
     def read_func(filename):
         return Table.read(filename), filename
 
     dat, info = chandra_models.get_data(
-        CHANDRA_MODELS_PITCH_ROLL_FILE, read_func=read_func
+        model_path, read_func=read_func, version=version
```

Comment:
I note it took me about an hour to figure out if the first argument to chandra_models.get_data is a full path, this mechanism silently breaks.  Copilot had suggested, for example, that ska_helpers.paths.chandra_models_repo_path made sense as the first part of the path to the model. And then at 

https://github.com/sot/ska_helpers/blob/9f72470107e83fc857a13115e1ba258acbdd9bb0/ska_helpers/chandra_models.py#L196

this just meant that I was consistently overriding the cloned-version repo with the full path to my git repo pitch roll file.  Not sure what makes sense as a ska_helpers chandra_models fix for that.